openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -233,18 +233,18 @@ def test_jwt_access_token_from_parameter_not_header(self, client_attr):
         (None, 'no_token_type_supplied'),
     )
     @ddt.unpack
-    @patch('edx_django_utils.monitoring.set_custom_metric')
-    def test_access_token_metrics(self, token_type, expected_token_type, mock_set_custom_metric):
+    @patch('edx_django_utils.monitoring.set_custom_attribute')
+    def test_access_token_metrics(self, token_type, expected_token_type, mock_set_custom_attribute):
```

Comment:
Rename test methods?

Reply:
Good finds. Mind if I pick those up in a second PR so that this can be a clean replacement?

Reply:
Go for it.  Do you want to start that PR now based off this branch?

Reply:
Oh, I was thinking of just merging this one and starting a new branch to keep things simple (since this one is going to get squashed).

Reply:
...but I'm not merging this until Monday anyhow, so I might as well start there and rebase later. :-)